### ISSUE TITLE
"_TweakAccessorGenerator will damage your computer" pop-up

### ISSUE THREAD

**Issue by mauronate** [justeat/JustTweak#61]
HI.
Since we updated to the 7.0.0 pod release, we are not able to run anymore the TweakAccessorGenerator, since we are getting this weird macOS pop up, saying that the executable is going to "damage your computer".
Eventually the pop-up stops appearing but the process now gets killed immediately with an error code of 9.

The weird thing is that, we tried to revert the pod to the 6.2.0 version, which we never had a problem with before the 7.0.0 update, but now it gives us the same Killed: error 9 message.

I see that there is probably already a pull request here: https://github.com/justeat/JustTweak/pull/60 addressing this issue, however I wondered if there was anyway to circumvent the problem in the meantime, since reverting to 6.2.0 seems to not have been enough.

**Comment by albertodebortoli** [comment]
Hi @mauronate, yes the problem is weird and surprised us today as it started happening all of a sudden.
PR #60, that you've already seen, should fix the issue but now, GitHub Actions fails
```
Fetching git@github.com:apple/swift-argument-parser.git

xcodebuild: error: Could not resolve package dependencies:
  Authentication failed because the credentials were missing
```

even though in #59 it went totally fine.

It's great to see JustTweak been used, so I've merged #60 and published 7.0.1 to trunk.
We'll address the issue with GitHub Actions later.

Please give it a try and see if that solves it for you, it seems that it did for us.

**Comment by mauronate** [comment]
Tried the 7.0.1 trunk right now, and the JustTweak generator script run without a problem. Thank you a lot!
